src/utlis/utils.js

```diff
@@ -25,3 +25,14 @@ function closeModalOnRemoteClick(evt) {
     closePopup(evt.target.closest(".modal"));
   }
 }
+
+export function toggleSaving(modalId) {
+  const modal = document.querySelector(modalId);
+  const saveButton = modal.querySelector(".form__popup-button");
+
+  if (saveButton.innerText === "Saving...") {
+    saveButton.innerText = "Save";
```

Comment:
Please don't use `innerText` for updating element contents, it's not secure. Use `textContent` instead 

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#:~:text=with%20new%20content.-,Note,-%3A%20This%20is

Reply:
updated